### PR TITLE
{Network} Regenerate aaz code

### PR DESCRIPTION
**Pull request by kairu-ms**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

**Review by necusjz — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/asg/_delete.py [hunk 1]
```diff
@@ -12,7 +12,8 @@
 
 
 @register_command(
-    "network asg delete"
+    "network asg delete",
+    confirmation="Are you sure you want to perform this operation?",
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/asg/_delete.py:16]: It seem the command model is not correct

File: src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/lb/probe/_create.py [hunk 2]
```diff
@@ -87,7 +87,6 @@ def _build_arguments_schema(cls, *args, **kwargs):
             options=["--probe-threshold"],
             arg_group="Properties",
             help={"short-summary": "The number of consecutive successful or failed probes in order to allow or deny traffic from being delivered to this endpoint. It is currently in preview and is not recommended for production workloads. For most scenarios, we recommend maintaining the default value of 1 by not specifying the value of the property.", "long-summary": "After failing the number of consecutive probes equal to this value, the endpoint will be taken out of rotation and require the same number of successful consecutive probes to be placed back in rotation."},
-            is_preview=True,
         )
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/lb/probe/_create.py:90]:
```suggestion
            is_preview=True,
        )
```

File: src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/lb/probe/_update.py [hunk 3]
```diff
@@ -88,7 +88,6 @@ def _build_arguments_schema(cls, *args, **kwargs):
             options=["--probe-threshold"],
             arg_group="Properties",
             help={"short-summary": "The number of consecutive successful or failed probes in order to allow or deny traffic from being delivered to this endpoint. It is currently in preview and is not recommended for production workloads. For most scenarios, we recommend maintaining the default value of 1 by not specifying the value of the property.", "long-summary": "After failing the number of consecutive probes equal to this value, the endpoint will be taken out of rotation and require the same number of successful consecutive probes to be placed back in rotation."},
-            is_preview=True,
             nullable=True,
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/lb/probe/_update.py:91]:
```suggestion
            is_preview=True,
            nullable=True,
```

File: src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/route_filter/rule/_delete.py [hunk 4]
```diff
@@ -13,7 +13,8 @@
 
 @register_command(
     "network route-filter rule delete",
-    is_preview=True
+    is_preview=True,
+    confirmation="Are you sure you want to perform this operation?",
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/route_filter/rule/_delete.py:17]:
```suggestion
    is_preview=True
```

File: src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/service_endpoint/policy/_delete.py [hunk 5]
```diff
@@ -13,6 +13,7 @@
 
 @register_command(
     "network service-endpoint policy delete",
+    confirmation="Are you sure you want to perform this operation?",
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/service_endpoint/policy/_delete.py:16]:
```suggestion
```

File: src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/route_table/route/_delete.py [hunk 6]
```diff
@@ -13,6 +13,7 @@
 
 @register_command(
     "network route-table route delete",
+    confirmation="Are you sure you want to perform this operation?",
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/route_table/route/_delete.py:16]:
```suggestion
```

File: src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/security_partner_provider/_delete.py [hunk 7]
```diff
@@ -13,7 +13,7 @@
 
 @register_command(
     "network security-partner-provider delete",
-    is_preview=True,
+    is_preview=True
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/security_partner_provider/_delete.py:16]:
```suggestion
    is_preview=True,
```

File: src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/service_endpoint/policy_definition/_delete.py [hunk 8]
```diff
@@ -13,6 +13,7 @@
 
 @register_command(
     "network service-endpoint policy-definition delete",
+    confirmation="Are you sure you want to perform this operation?",
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/service_endpoint/policy_definition/_delete.py:16]:
```suggestion
```

**Pull request merged**